src/credential_offer.rs

```diff
@@ -137,4 +202,41 @@ mod test {
         }))
         .unwrap();
     }
+
+    #[test]
+    fn example_credential_offer_object_value() {
+        let _: CredentialOfferParameters<CoreProfilesOffer> = serde_json::from_value(json!({
+            "credential_issuer": "https://credential-issuer.example.com",
+            "credentials": [{
+                "format": "jwt_vc_json-ld",
+                "credential_definition": {
+                    "@context": [
+                        "https://www.w3.org/2018/credentials/v1",
+                        "https://www.w3.org/2018/credentials/examples/v1"
+                    ],
+                    "type": [
+                        "VerifiableCredential",
+                        "UniversityDegreeCredential"
+                    ]
+                }
+            }, {
+                "format": "mso_mdoc",
+                "doctype": "org.iso.18013.5.1.mDL",
+            }],
+            "grants": {
+                "authorization_code": {
```

Comment:
Do you know if it's OK to have multiple grants like this? Having both authorization code and pre-authorized code seems a bit jarring

Reply:
After looking at the spec again, it seems to be allowed:
>  grants: OPTIONAL. Object indicating to the Wallet the Grant Types the Credential Issuer's Authorization Server is prepared to process for this Credential Offer. Every grant is represented by a name/value pair. The name is the Grant Type identifier; the value is an object that contains parameters either determining the way the Wallet MUST use the particular grant and/or parameters the Wallet MUST send with the respective request(s). If grants is not present or is empty, the Wallet MUST determine the Grant Types the Credential Issuer's Authorization Server supports using the respective metadata. **When multiple grants are present, it is at the Wallet's discretion which one to use.**